Gemfile

```diff
@@ -15,6 +15,10 @@ gem 'dotenv-rails'
 gem 'rack-cors', :require => 'rack/cors'
 gem 'aasm'
 gem 'geocoder'
+#the regular private pub would not work.  Had to create own version of it and push it to ruby gems
+#due to validation of time stamp and sig.
+gem 'private_pub_no_sig'
+gem "thin"
```

Comment:
I had to implement my own version of private pub because we were using private pub across platforms and I couldn't figure out the encoding of signature so I had to remove the check in the gem.  This way it won't validate the signature or the timestamp of the subscription.

my version of the gem:  https://github.com/teelu/private_pub